js/ui/editor/editor.js

```diff
@@ -303,15 +331,16 @@ var Editor = Widget.inherit({
                 this._refreshFocusState();
                 break;
             case "value":
-                if(!this._valueChangeActionSuppressed) {
-                    this._raiseValueChangeAction(args.value, args.previousValue);
-                    this._saveValueChangeEvent(undefined);
-                }
-                if(args.value != args.previousValue) { // jshint ignore:line
-                    this.validationRequest.fire({
-                        value: args.value,
-                        editor: this
-                    });
+                var argument;
+                window.clearTimeout(that._valueChangeTimeout);
+
+                if(args.value !== "" && that.option("updateValueTimeout")) {
+                    argument = args;
+                    that._valueChangeTimeout = window.setTimeout(function() {
```

Comment:
So you **always** wrap `_valueOptionChange` into timeout for a base editor? I don't think that it's a good idea for common scenarios.

Reply:
No, I Wrap it only if timeout is set.